### PR TITLE
Pn/tickets endpoint

### PR DESCRIPTION
**Pull request by Arkonzoa**
#PR Content
 - Past tickets
    - Changed the path to /tickets/user/:id/past
    - Updated the doc to take into account the path changes
    - Added the filter to show only those that are before the current day
  - Upcoming tickets
    - Changed path to /tickets/user/:id/past
    - Updated the doc to take into account the path changes
    - Added a filter to show only the tickets that are after the current day

### CODE REVIEW

File: express-server/src/api/tickets/controller.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { success, notFound, authorOrAdmin } from '../../services/response/'
 import { Tickets } from '.'
+var ObjectId = require('mongodb').ObjectId;
```
Comment (Arkonzoa) [inline express-server/src/api/tickets/controller.js]: Need this to cast is as an ObjectId to compare it against the auto-generated IDs in the documents

Comment (cesar3030) [reply express-server/src/api/tickets/controller.js]: That weird I've never had to use that shit.

Comment (Arkonzoa) [reply express-server/src/api/tickets/controller.js]: It's used when I'm looking up a user through a ref document

File: express-server/src/api/tickets/index.js [hunk 2]
```diff
@@ -290,11 +290,12 @@ router.delete('/:id',
   destroy)
 
 /**
- * @api {get} /tickets/past Retrieve past tickets
+ * @api {get} /tickets/user/:id/past Retrieve past tickets
```
Comment (cesar3030) [inline express-server/src/api/tickets/index.js]: I'm not sure why we are doing that. The first version was correct.
The mobile app needs to get tickets of the connected user only witch we have the id in the token.
This endpoint is ok if we want to retrieve tickets of a user that is not the one connected and I don't see any specs asking for that behavior.

If I want to see my past events on mobile I should do /tickets/past and not /tickets/<my_id>/past

File: express-server/src/api/tickets/model.js [hunk 3]
```diff
@@ -34,7 +34,7 @@ ticketsSchema.methods = {
       user: this.user.view(full),
       price: this.price,
       deleted_at: this.deleted_at,
-      event: this.event,
+      event: this.event.view(full),
```
Comment (cesar3030) [inline express-server/src/api/tickets/model.js]: Genial! je me demandais comment faire ca! Thanks man

**Comment by cesar3030** [comment]
Est ce que upcoming fonctionne au final?

**Comment by Arkonzoa** [comment]
Upcoming et past fonctionne maintenant, il ne me restera qu'à retravailler create

**Pull request merged**